gradio/components/image.py

```diff
@@ -138,19 +138,28 @@ def __init__(
 
         if isinstance(watermark, (str, Path, PIL.Image.Image, np.ndarray)):
             warnings.warn(
-                "The `watermark` parameter is updated to use WatermarkOptions. Please use the `watermark` parameter with a `gr.WatermarkOptions` instance instead."
+                "Passing a string, Path, PIL.Image, or numpy array to 'watermark' will be removed in Gradio 6.0. "
```

Comment:
This is actually not done yet. Passing in a PIL.image will not cause an error I believe.